src/bip/plugins/tango/signal_data_packet.py

```diff
@@ -116,7 +116,7 @@ def add_record(self,
 
             "stream_id": np.uint32(packet.stream_id),
             "sample_count": np.uint32(packet.sample_count),
-            "trailer": np.uint32(packet.trailer),
+            "trailer": packet.trailer,
```

Comment:
Do you not need to use the np.uint32 anymore because the schema already makes sure that it is a uint32 type in the list?

Reply:
packet.trailer is a list and lists cannot be converted into np.uint32.
The list itself is created in /src/bip/vita/signal_data_packet.py.
The schema makes sure that the list only contains values with dtype= uint32